numpy/distutils/misc_util.py

```diff
@@ -2225,7 +2224,6 @@ def is_bootstrapping():
         return True
     except AttributeError:
         return False
-        __NUMPY_SETUP__ = False
```

Comment:
Pretty sure this one is important, and used by another file.

Reply:
It is local to the function and can have no effect. Looks like a cut and paste refactor bug.